movement/movement.c

```diff
@@ -476,7 +483,13 @@ bool app_loop(void) {
 
     event.subsecond = 0;
 
-    return can_sleep && (movement_state.light_ticks == -1) && !movement_state.is_buzzing;
+    // if the watch face changed, we can't sleep because we need to update the display.
```

Comment:
Nice, this whole section is now much clearer. For 'consistency', I guess we could remove the 'return false' after move_to_next_face in simple_clock_face, world_clock_face, and preferences_face.

Reply:
I don't think it matters tbh; I added [documentation](https://github.com/joeycastillo/Sensor-Watch/pull/94/commits/f89cecc56a974b914b54a190157bd77d1e7c9be9) noting that the return value is moot once you call `movement_move_to_next_face`.